ssh/tunnel.py

```diff
@@ -16,8 +16,6 @@
 
 from pkgpanda.util import write_string
 
-LOGGING_FORMAT = '[%(asctime)s|%(name)s|%(levelname)s]: %(message)s'
```

Comment:
Yes. It is a very good idea to have a common log configuration

The only files that should be setting the logging format are ones that:
- Really specifically need their own format (I cannot see where this would happen curently)
- Files that format and setup the logging for the current session.

Furthermore, when a module chooses to format logging, it should do so at runtime, because when you do it at import time like this, you have no idea if this format will actually be used anywhere. This module specifically gets its logger from logging.getLogger() so its clear that it should inherit the format from whoever is using it.

So, this PR doesnt force user to have a common logger. Rather it allows a user to have a modular logger where settings can be inherited and configured dynamically or per module. Having a common logger would be adding a new file to this module called 'logging.py' which defines how logging should be setup globally and then every file then imports it.